src/lib/utils.ts

```diff
@@ -92,47 +77,21 @@ export const shortenUrl = (
  * @param inputBytes
  * @returns string
  */
-export function bytesToiB(inputBytes: number, isBinary: boolean): string {
-  const options: {
-    preferByte: boolean
-    preferBinary: boolean
-    preferDecimal: boolean
-  } = {
-    preferByte: true,
-    preferBinary: isBinary,
-    preferDecimal: !isBinary,
-  }
-  let autoscale = byteConverter.autoScale(
-    inputBytes,
-    'B',
-    options as ByteConverterAutoscaleOptions,
-  )
-  let stringVal = ''
-  if (autoscale.dataFormat === 'YiB') {
-    autoscale = byteConverter.autoScale(
-      inputBytes - 32,
-      'B',
-      options as ByteConverterAutoscaleOptions,
-    )
-    return `${autoscale.value.toFixed(1)}${autoscale.dataFormat}`
+export function bytesToiB(inputBytes: number): string {
+  try {
+    const parsedValue = bytes(inputBytes, { mode: 'binary' })
+    return parsedValue ?? '0GiB'
```

Comment:
When can `parsedValue` be empty?
I think empty parsedValue should be treated as an error - so even if we fallback to `0 GiB` we should log that it happened.